src/Entities/Hand.ts

```diff
@@ -1,20 +1,60 @@
-import Card from "./Card";
+import Card from './Card'
 
 class Hand {
-  private hand: Card[];
+  private hand: Card[]
 
   constructor() {
-    this.hand = [];
+    this.hand = []
   }
 
-  public getCard(cardId: string): Card {
-    // find the card that has that cardId and then remove and return it
-    throw Error("get card not implemented");
+  public addCard(card: Card): void {
+    this.hand.push(card)
```

Comment:
I thought about making a prevision to ask when a new card is added if it is already in the `Hand` and in that case throw an error because it shouldn't be possible. I decided not to because I know there are games that use more than on deck so for now I am going to leave it not checking.

Reply:
whatever :/